comet-java-client/src/test/java/ml/comet/experiment/impl/OnlineExperimentTest.java

```diff
@@ -323,25 +388,48 @@ public void testLogAndGetExperimentTime() {
 
     @Test
     public void testUploadAndGetAssets() {
-        OnlineExperiment experiment = createOnlineExperiment();
+        OnlineExperimentImpl experiment = (OnlineExperimentImpl) createOnlineExperiment();
 
+        // Make sure experiment has no assets
```

Comment:
when the action is very strait forward like x.isEmpty no need to document (its self explained) 

comments / documents are good on complex logic on non trivial assertion / setup